proto/src/main/proto/Rss.proto

```diff
@@ -260,6 +260,9 @@ service CoordinatorServer {
 
   // Access to the remote shuffle service cluster
   rpc accessCluster(AccessClusterRequest) returns (AccessClusterResponse);
+
+  // Get basic client conf from coordinator
+  rpc fetchClientConf(google.protobuf.Empty) returns (FetchClientConfResponse);
```

Comment:
Maybe we should a request which tell us the source.From azkanban,livy,us.

Reply:
Ajdust processing logic according to the platform may be needed in the future but no the scope of this pr.